### DISCUSSION TITLE
Github link top right shows tag ?

### DISCUSSION THREAD

**Discussion by TRaSH-** [squidfunk/mkdocs-material#4189] [Q&A] [answered]
I had to add a tag to my github page for something else.
and i noticed later today that it now shows in the top right the tag name and not any more the starred and forks etc.

![image](https://user-images.githubusercontent.com/6155095/182003797-ce4f6206-45b8-4d3b-8614-460f5627e315.png)

any way to change this back or edit something ?

**Comment by squidfunk** [comment, marked as answer]
Material for MkDocs will automatically query the latest release and show it. You can hide it with extra CSS, ~~see https://github.com/squidfunk/mkdocs-material/issues/2481#issuecomment-804046887~~.

__Edit__: The CSS is even simpler after the last refactoring:

``` css
.md-source__fact--version {
  display: none;
}
```

**Reply by TRaSH-** [reply]
thnx
